llm/llama-3_1/README.md

```diff
@@ -0,0 +1,306 @@
+# Serve Llama 3.1 on Your Own Infrastructure
+
+
+<p align="center">
+<img src="https://i.imgur.com/kQGzHI6.png" alt="Llama-3.1 on SkyPilot" style="width: 70%;">
+</p>
+
+On July 23, 2024, Meta AI released the [Llama 3.1 model family](https://ai.meta.com/blog/meta-llama-3-1/), including a 405B parameter model in both base model and instruction-tuned forms.
+
+Llama 3.1 405B became the most capable open LLM model to date. This is **the first time an open LLM closely rivals state-of-the-art proprietary models** like GPT-4o and Claude 3.5 Sonnet.
+
+This guide walks through how to serve Llama 3.1 models **completely on your infrastructure** (cluster or cloud VPC). Supported infra:
+
+- Local GPU workstation
+- Kubernetes cluster
+- Cloud accounts ([12 clouds supported](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html))
+
+SkyPilot will be used as the unified framework to launch serving on any (or multiple) infra that you bring.
+
+## Serving Llama 3.1 on your infra
+
+We will first test the model on a GPU dev node, then package it for deployment using SkyPilot.
```

Comment:
Can we have highlighted sentence for the complete deployment YAML so people can directly jump to that section? 
```suggestion
We offer a step-by-step guide for how to use SkyPilot to test a new model on a GPU dev node, and then package it for deployment using SkyPilot. **For the complete deployment guide of Llama 3.1, see [Step 3: Package and deploy using SkyPilot](#Step-3-Package-and-deploy-using-SkyPilot)
```